test/commands/solution/search_test.rb

```diff
@@ -69,4 +69,31 @@ class Solution::SearchTest < ActiveSupport::TestCase
     assert_equal [completed], Solution::Search.(user, mentoring_status: :completed)
     assert_equal [completed], Solution::Search.(user, mentoring_status: 'completed')
   end
+
+  test "pagination" do
+    user = create :user
+    solution_1 = create :concept_solution, user: user
+    solution_2 = create :concept_solution, user: user
+
+    assert_equal [solution_1], Solution::Search.(user, page: 1, per: 1)
+    assert_equal [solution_2], Solution::Search.(user, page: 2, per: 1)
+    assert_equal [solution_1, solution_2], Solution::Search.(user, page: 1, per: 2)
+    assert_equal [], Solution::Search.(user, page: 2, per: 2)
+  end
+
+  test "sort oldest first" do
+    user = create :user
+    solution_1 = create :concept_solution, user: user, created_at: 2.days.ago
+    solution_2 = create :concept_solution, user: user, created_at: 1.day.ago
+
+    assert_equal [solution_1, solution_2], Solution::Search.(user, sort: "oldest_first")
+  end
+
+  test "sort newest first" do
+    user = create :user
+    solution_1 = create :concept_solution, user: user, created_at: 1.day.ago
+    solution_2 = create :concept_solution, user: user, created_at: 2.days.ago
```

Comment:
Maybe indicate which solution is older in the variable names?

```suggestion
    new_solution = create :concept_solution, user: user, created_at: 1.day.ago
    old_solution = create :concept_solution, user: user, created_at: 2.days.ago
```